infrastructure-playbooks/rolling_update.yml

```diff
@@ -194,6 +194,17 @@
                   when: (check_quorum_status.stdout | from_json).quorum | length != groups[mon_group_name] | length
               when: inventory_hostname == groups[mon_group_name] | first
 
+        - name: Replace systemd docker files with updates
```

Comment:
maybe we can change this from "docker files" to "unit" and have something like:

"Replace ceph-mon systemd unit"

Just saying "docker files" is a bit weird, but not blocking for this.